src/models/RekognitionModel.ts

```diff
@@ -11,8 +12,8 @@ export default class RekognitionModel extends BaseModel {
     this.rekognition = rekognitionConnection.getClient();
   }
 
-  detectFaces (imageIds: string[]) {
-    return;
+  detectFaces (image: string) {
+    return mockResponse;
```

Comment:
just to make sure that I understand, whatever image we pass in we get the first mock response, correct?